packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md

```diff
@@ -164,20 +162,20 @@ For testing purposes, use [this tool](https://www.jsonwebtoken.dev) to generate
 
 You can use the following [JWT claims](/docs/reference/api/oidc/#token-claims-for-client-authentication-with-client-secret-or-private-key-jwt) in the request for a scoped access token:
 
-* `alg` &mdash; one of the supported algorithm values: RS265, RS384, RS512, ES256, ES384, or ES512. This is required for Okta to successfully verify the token using the signing keys provided in the [previous step](#create-a-service-app-and-grant-scopes). The `alg` parameter goes in the header of the JWT rather than a claim in the payload of the body.
-* `aud` &mdash; the full URL of the resource that you're using the JWT to authenticate to
-* `exp` &mdash; the expiration time of the token in seconds since January 1, 1970 UTC (current UNIX timestamp). This value must be a maximum of only an hour in the future.
-* `jti` &mdash; (Optional) A unique identifier of the token. This value is used to prevent the JWT from being replayed. The claim is a case-sensitive string.
-* `iat` &mdash; (Optional) the issuing time of the token in seconds since January 1, 1970 UTC (current UNIX timestamp)
-* `iss` &mdash; the issuer of the token. This value must be the same as the `client_id`.
-* `sub` &mdash; the subject of the token. This value must be the same as the `client_id`.
+* `alg`: One of the supported algorithm values: RS265, RS384, RS512, ES256, ES384, or ES512. This is required for Okta to successfully verify the token using the signing keys provided in the [previous step](#create-a-service-app-and-grant-scopes). The `alg` parameter goes in the header of the JWT rather than a claim in the payload of the body.
+* `aud`: The full URL of the resource that you're using the JWT to authenticate to
+* `exp`: The expiration time of the token in seconds since January 1, 1970 UTC (current UNIX timestamp). This value must be a maximum of only an hour in the future.
+* `jti`: (Optional) A unique identifier of the token. This value is used to prevent the JWT from being replayed. The claim is a case-sensitive string.
```

Comment:
```suggestion
* `jti`: (Optional) The token's unique identifier. This value is used to prevent the JWT from being replayed. The claim is a case-sensitive string.
```